### PR TITLE
nameChanges

### PR DESCRIPTION
**Pull request by simran7269**

### CODE REVIEW

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -11,7 +11,15 @@ import { WishListComponent } from './wish-list/wish-list.component';
 
 const routes: Routes = [
   { path:'dashboard',
-component:DashboardComponent},
+component:DashboardComponent,
+
+// children:[
```
Comment (Sushmitha-Vk) [inline src/app/app-routing.module.ts]: remove commented code

File: src/app/card/card.component.ts [hunk 2]
```diff
@@ -27,15 +27,15 @@ value: any;
...
         
   
 
   }
   delete(id:any) {
       
-    this.service.deleteOrder(id).subscribe(d=>
-      {this.router.navigate(["dashboard"])});
+    this.service.deleteOrder(id).subscribe()
+      {this.router.navigate(["dashboard"])};
```
Comment (Sushmitha-Vk) [inline src/app/card/card.component.ts]: this is not the right way, router navigate should come inside subscribe

File: src/app/dashboard/dashboard.component.ts [hunk 3]
```diff
@@ -79,18 +77,18 @@ export class DashboardComponent implements OnInit {
...
-  xyz(id:any){
+  allRow(id:any){
       this.router.navigate(['card/',id])
     }
 
 
     star(data: any) {
       
-      this.service.updateFavorites(data).subscribe((d: any) => {
+      this.service.updateFavorites(data).subscribe()
```
Comment (Sushmitha-Vk) [inline src/app/dashboard/dashboard.component.ts]: reload should come inside subscribe only

File: src/app/login-page/login-page.component.ts [hunk 4]
```diff
@@ -34,12 +34,12 @@ export class LoginPageComponent implements OnInit {
 
   
   login(data: any) {
-    this.serv.login(data).subscribe(d => {
+    this.serv.login(data).subscribe()
```
Comment (Sushmitha-Vk) [inline src/app/login-page/login-page.component.ts]: this is wrong, how come you'll move the below 3 lines of code outside subscribe, there is a reason why we wrote those 3 lines inside subscribe, explain to me tomorrow why we have written like that

File: src/app/order-details/order-details.component.ts [hunk 5]
```diff
@@ -33,22 +33,22 @@ export class OrderDetailsComponent implements OnInit {
   }
 
   submit(data: any) {
-    this.serv.createOrder({ ...data, id: data['sno'] }).subscribe(d=>{
-
```
Comment (Sushmitha-Vk) [inline src/app/order-details/order-details.component.ts:37]: same here

**Review by Sushmitha-Vk — CHANGES_REQUESTED** [review]

File: src/app/app-routing.module.ts (hunk 1 above)
Comment (simran7269) [reply src/app/app-routing.module.ts]: resolved

File: src/app/card/card.component.ts (hunk 2 above)
Comment (simran7269) [reply src/app/card/card.component.ts]: resolved

File: src/app/dashboard/dashboard.component.ts (hunk 3 above)
Comment (simran7269) [reply src/app/dashboard/dashboard.component.ts]: resolved

File: src/app/login-page/login-page.component.ts (hunk 4 above)
Comment (simran7269) [reply src/app/login-page/login-page.component.ts]: resolved

File: src/app/order-details/order-details.component.ts (hunk 5 above)
Comment (simran7269) [reply src/app/order-details/order-details.component.ts:37]: resolved

File: src/app/card/card.component.ts [hunk 6]
```diff
@@ -9,47 +10,55 @@ import { SampleserviceService } from '../sampleservice.service';
...
+    });
   }
 
   ngOnInit() {
-    
-    this.service.getDetails(this.route.snapshot.params['id']).subscribe(x=>{
-      this.idetails=x
-      
+
+    this.service.getDetails(this.route.snapshot.params['id']).pipe(takeUntil(this.destroyedValues$)).subscribe(x => {
```
Comment (Sushmitha-Vk) [inline src/app/card/card.component.ts]: why again x...?

File: src/app/card/card.component.ts [hunk 7]
```diff
@@ -9,47 +10,55 @@ import { SampleserviceService } from '../sampleservice.service';
...
-  delete(id:any) {
-      
-    this.service.deleteOrder(id).subscribe(d=>
-      {this.router.navigate(["dashboard"])});
+  delete(id: any) {
 
+    this.service.deleteOrder(id).pipe(takeUntil(this.destroyedValues$)).subscribe(() => {
 
+
+      { this.router.navigate(["dashboard"]) };
```
Comment (Sushmitha-Vk) [inline src/app/card/card.component.ts]: why this {} for router.navigate

File: src/app/dashboard/dashboard.component.ts [hunk 8]
```diff
@@ -79,18 +77,21 @@ export class DashboardComponent implements OnInit {
...
+
+      
        
 
         window.location.reload();
       });
+      }
 
      
   // logout() {
```
Comment (Sushmitha-Vk) [inline src/app/dashboard/dashboard.component.ts]: remove commented

File: src/app/order-details/order-details.component.ts [hunk 9]
```diff
@@ -33,22 +36,24 @@ export class OrderDetailsComponent implements OnInit {
...
 
 
 
-    this.dialog.closeAll();
-     });
-  }
   update(value: any){
-     this.serv.editDetails(value).subscribe(a =>{
+     this.serv.editDetails(value).pipe(takeUntil(this.destroyedData$)).subscribe()
       // window.location.reload();
```
Comment (Sushmitha-Vk) [inline src/app/order-details/order-details.component.ts:54]: remove all commented lines

**Review by Sushmitha-Vk — CHANGES_REQUESTED** [review]

File: src/app/card/card.component.ts (hunk 6 above)
Comment (simran7269) [reply src/app/card/card.component.ts]: resolved

File: src/app/dashboard/dashboard.component.ts (hunk 8 above)
Comment (simran7269) [reply src/app/dashboard/dashboard.component.ts]: resolved

**Review by Sushmitha-Vk — APPROVED** [review]

**Pull request merged**